nucypher/network/server.py

```diff
@@ -113,56 +113,57 @@ def _make_rest_app(datastore: Datastore, this_node, domain: str, log: Logger) ->
 
     @rest_app.route("/public_information")
     def public_information():
-        """
-        REST endpoint for public keys and address.
-        """
-        response = Response(
-            response=bytes(this_node),
-            mimetype='application/octet-stream')
-
+        """REST endpoint for public keys and address."""
+        response = Response(response=bytes(this_node), mimetype='application/octet-stream')
         return response
 
-    @rest_app.route("/ping", methods=['POST'])
+    @rest_app.route("/ping", methods=['GET', 'POST'])
     def ping():
         """
-        Asks this node: "Can you access my public information endpoint"?
+        GET: Asks this node: "What is my IP address?"
+        POST: Asks this node: "Can you access my public information endpoint?"
         """
 
-        try:
-            requesting_ursula = Ursula.from_bytes(request.data)
-            requesting_ursula.mature()
-        except ValueError:  # (ValueError)
-            return Response({'error': 'Invalid Ursula'}, status=400)
-        else:
-            initiator_address, initiator_port = tuple(requesting_ursula.rest_interface)
-
-        # Compare requester and posted Ursula information
-        request_address = request.environ['REMOTE_ADDR']
-        if request_address != initiator_address:
-            return Response({'error': 'Suspicious origin address'}, status=400)
+        if request.method == 'GET':
+            requester_ip_address = request.environ['REMOTE_ADDR']
```

Comment:
Why not `request.remote_addr`?

Reply:
I'm looking for documentation or good reference on this, but cannot seem to find it :thinking:

Reply:
There's https://flask.palletsprojects.com/en/1.1.x/api/#flask.Request.remote_addr for `remote_addr`.

Reply:
:heavy_check_mark: - Thanks!